docs/python/inference/api_summary.rst

```diff
@@ -265,7 +265,7 @@ Internal classes
 ----------------
 
 These classes cannot be instantiated by users but they are returned
-by methods or functions of this libary.
+by methods or functions of this library.
```

Comment:
@xadupre Can you confirm this file can be directly modified?

Reply:
Yes, every file .rst in the repo can be manually modified. If their are not in the repo, they are automatically generated.